src/server.js

```diff
@@ -69,6 +64,8 @@ const startServer = async (portNumber) => {
     app.delete("/api/session", logoutUser);
     app.post("/api/passwordReset", userResetPassword);
 
+    app.get("/:database", postgres);
```

Comment:
`postgres` router can't render postgres page because `postgres.ejs` is missing. I think you changed the name to `database.ejs`